bin/cloudgov/events/get-events

```diff
@@ -65,17 +65,26 @@ SPACE_GUID=$(cf space $SPACE --guid)
 cf target -s $SPACE &>/dev/null
 
 declare -A SPACE_GUIDS=()
-# Keep the code needed to query multiple spaces for events. Not working now, but may be possible.
-#  for SPACE in dev stage prod shared-egress shared-egress-dr tools; do
-for SPC in $SPACE; do
+
+# Attempt to fetch uuids for all usagov cf spaces:
+SPACE_QUERY=''
+for SPC in dr dev stage prod shared-egress tools; do
+#for SPC in $SPACE; do
   if [ ! cf space $SPC &> /dev/null ]; then
     echo "ERROR: CF space '$SPC' does not exist!"
   else
     SPACE_GUID=$(cf space $SPC --guid)
     SPACE_GUIDS[$SPACE_GUID]=$SPC
+    if [ -z $SPACE_QUERY ]; then
+      SPACE_QUERY=$SPACE_GUID
+    else
+      SPACE_QUERY="$SPACE_QUERY,$SPACE_GUID"
+    fi
   fi
 done
 
+echo "NOTICE: SPACE_QUERY: $SPACE_QUERY"
```

Comment:
Okay, this is something you'll want to remove before we merge this code in, I think.